src/core/macros.systemd.in

```diff
@@ -122,9 +122,9 @@ SYSTEMD_INLINE_EOF \
 #   %{_sysusersdir}/%{name}.conf
 %sysusers_create_package() \
 %{expand:%%{?!__systemd_twoargs_%#:%%{error:This macro requires two arguments}}} \
-systemd-sysusers --replace=%_sysusersdir/%1.conf - <<SYSTEMD_INLINE_EOF >/dev/null 2>&1 || : \
+systemd-sysusers --replace=%_sysusersdir/%1.conf - <<SYSTEMD_INLINE_EOF || : \
```

Comment:
I'm not sure if we should wrap this in `if which systemd-sysusers &>/dev/null; then ... fi`. On the one hand, people mostly call this when they need the users to exists. On the other hand, maybe sometimes they don't. I guess we can adjust if people complain ;)

Reply:
Creating users in such a strange way makes sense only when they are sctrictly required before installing a package, when some files in packages are owned by that user (`%attr(-,user,group)`). In this situation such wrapping is not needed to my mind. And I would you POSIX shell compatible `if command -v systemd-sysusers`.